dynamic-rapidpro-workspace-settings/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "update-rapidpro-workspace-settings",
+  "version": "1.0.0",
+  "description": "dynamic-rapidpro-workspace Shared GitHub Action",
+  "main": "index.js",
+  "scripts": {
+    "build": "ncc build index.js --license licenses.txt",
+    "test": "mocha test/*.spec.js --timeout 10000 --slow 500"
+  },
+  "author": "Medic Inc.",
+  "license": "ISC",
+  "dependencies": {
+    "@actions/core": "^1.5.0",
+    "@actions/github": "^5.0.0",
+    "axios": "^0.21.4",
+    "chai": "^4.3.6",
+    "fs": "^0.0.1-security",
+    "mocha": "^9.2.1",
```

Comment:
mocha and chai should be dev dependencies

Reply:
Moved.